drizzle-orm/src/mysql-core/columns/custom.ts

```diff
@@ -57,12 +57,15 @@ export class MySqlCustomColumnBuilder<T extends ColumnBuilderBaseConfig<'custom'
 	}
 }
 
+export type AnyMySqlCustomColumn = MySqlCustomColumn<any>;
+
 export class MySqlCustomColumn<T extends ColumnBaseConfig<'custom', 'MySqlCustomColumn'>> extends MySqlColumn<T> {
 	static readonly [entityKind]: string = 'MySqlCustomColumn';
 
 	private sqlName: string;
 	private mapTo?: (value: T['data']) => T['driverParam'];
 	private mapFrom?: (value: T['driverParam']) => T['data'];
+	customSelect?: (value: MySqlColumn | string | Name) => SQL;
```

Comment:
In my opinion, line 213, `value` should only be `MySqlColumn` because it's the user facing side. The API should just expect the user to pass the column to the `customSelect`.
I was tempted to do `value: any` for line 68 because that's internal to drizzle, when I was prototyping I wasn't sure yet what parameters I actually needed to pass to this function.
Why do you think `ColumnBuilderBaseConfig` and `ColumnTypeConfig` need a new property? I don't think any of the classes using these configurations need a new property.

Reply:
You're right, those types are just for config, they don't need to be changed. 
I thought that they contained types used in the class many times in order to re-use them and not have to declare them multiple times. But they are not